solidity/contracts/Oracle.sol

```diff
@@ -118,11 +118,14 @@ contract Oracle is Ownable {
     withdrawableWei = oneForConsistentGasCost;
   }
 
-  function cancel(bytes32 _externalId) public {
+  function cancel(bytes32 _externalId)
+    public
+    returns (bool)
+  {
     uint256 internalId = uint256(keccak256(msg.sender, _externalId));
     require(msg.sender == callbacks[internalId].addr);
     Callback memory cb = callbacks[internalId];
-    LINK.transfer(cb.addr, cb.amount);
+    require(LINK.transfer(cb.addr, cb.amount));
     delete callbacks[internalId];
```

Comment:
Does `delete` return a bool? Do we return true here? Before, success was not reverting.

Reply:
Just wanted to bump this question. How do we return a `bool` here?